### ISSUE TITLE
Website: Navigation tab order

### ISSUE THREAD

**Issue by ArakTaiRoth** [GovAlta/ui-components#1651]
# Info

Tab order in our website should be header --> left navigation --> right navigation (when it exists) --> content. Right now, it's header --> left navigation --> content --> right navigation.

## Acceptance Criteria:

1. Tab order goes in this order, header --> left navigation --> right navigation --> content

**Comment by ArakTaiRoth** [comment]
This can't be done until #1540 is complete

**Comment by Spark450** [comment]
Out of date or stale
